scripts/ci/test_source.py

```diff
@@ -40,7 +42,7 @@
     ado_branch_last_commit = os.environ.get('ADO_PULL_REQUEST_LATEST_COMMIT')
     ado_target_branch = os.environ.get('ADO_PULL_REQUEST_TARGET_BRANCH')
     if ado_branch_last_commit and ado_target_branch:
-        if ado_branch_last_commit == '$(System.PullRequest.SourceCommitId)':
+        if ado_branch_last_commit == 'HEAD':
```

Comment:
I don't think we should change this, as this comparison detects whether the `ado_branch_last_commit` is _not_ set.